packages/plugin-e2e/src/models/components/TimeRange.ts

```diff
@@ -25,7 +25,8 @@ export class TimeRange extends GrafanaPage {
         : this.ctx.page.getByRole('button', { name: 'Change time settings' });
 
       await changeTimeSettingsButton.click();
-      await this.getByGrafanaSelector(TimeZonePicker.containerV2).fill(zone);
+      await this.getByGrafanaSelector(TimeZonePicker.containerV2).click();
```

Comment:
Is this a bug related to plugin-e2e or has something changed in Grafana which means we need to apply a change here? Asking in case we need to guard against different versions of Grafana.

Reply:
I've tried this in Grafana 9.0 and 10.2 and the `zone` code doesn't seem to have ever worked. When I use this fix in Grafana 9 or 10.2 it works as expected. So this is a bug in plugin-e2e IMHO 👍